src/sort/set.rs

```diff
@@ -190,6 +190,11 @@ impl Sort for SetSort {
         }
         Some((cost, expr))
     }
+
+    fn serialized_name(&self, _value: &Value) -> Symbol {
+        "set-of".into()
+    }
+
```

Comment:
This is an unrelated change... It means that when set constructors are serialized they will use this as their name instead of the name of the sort.